PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -263,6 +263,19 @@ def miniAOD_customizeCommon(process):
     from RecoTauTag.Configuration.boostedHPSPFTaus_cfi import addBoostedTaus
     addBoostedTaus(process)
     #---------------------------------------------------------------------------
+    #Adding tau reco for 80X legacy reMiniAOD
+    #make a copy of makePatTauTask to avoid labels and substitution problems
+    _makePatTausTaskWithTauReReco = process.makePatTausTask.copy()
+    #add PFTau reco modules to cloned makePatTauTask
+    process.load("RecoTauTag.Configuration.RecoPFTauTag_cff")
+    from PhysicsTools.PatAlgos.tools.helpers import listModules
+    for module in listModules(process.PFTau):
+        _makePatTausTaskWithTauReReco.add(module)
```

Comment:
please define tasks and derive relevant sequences from them in RecoPFTauTag_cff so that here you can import the task and simply attach it instead of constructing it in place.
We should eventually migrate to that.